dbt_project.yml

```diff
@@ -34,5 +34,5 @@ models:
     +materialized: table # field history tables are configured to be incremental, except for int_jira__field_history_scd, which has to be a table
     +schema: jira
     intermediate:
-      +materialized: ephemeral 
+      +materialized: table 
```

Comment:
Does this need to be materialized as a table or can we make it ephemeral or a view?

Reply:
ack good catch i changed it to table when testing